java-api-core/src/com/kinvey/java/network/NetworkManager.java

```diff
@@ -1015,7 +1015,7 @@ public T execute() throws IOException {
     }
 
     /** used internally **/
-    enum SaveMode {
+    public enum SaveMode {
```

Comment:
if this is used internally (as per the code comment), why is the access set to `public`?

Reply:
Moved `SaveMode` enum class to separated file.